mobile-widgets/qmlmanager.cpp

```diff
@@ -1067,6 +1074,32 @@ void QMLManager::changesNeedSaving()
 	saveChangesCloud(false);
 #endif
 }
+
+void QMLManager::openNoCloudRepo()
+/*
+ * Open the No Cloud repo. In case this repo does not (yet)
+ * exists, create one first. When done, open the repo, which
+ * is obviously empty when just created.
+ */
+{
+	char *filename = NOCLOUD_LOCALSTORAGE;
+	const char *branch;
+	struct git_repository *git;
+
+	git = is_git_repository(filename, &branch, NULL, false);
+
+	if (git == dummy_git_repository) {
+		if (git_create_local_repo(filename))
+			appendTextToLog(get_error_string());
```

Comment:
if we get an error here, should we continue to save the filename as if all was ok?

Reply:
Agree, not as nice as it could be. I stole these lines from your code, which does no error handling here as well :-) And I believe: manageable risk. We will never get here when the disk is full or not writable or any other serious reason that the repo could not be created.